virtio-devices/src/epoll_helper.rs

```diff
@@ -47,6 +49,18 @@ pub trait EpollHelperHandler {
         helper: &mut EpollHelper,
         event: &epoll::Event,
     ) -> Result<(), EpollHelperError>;
+
+    fn handle_timeout(&mut self, _helper: &mut EpollHelper) -> Result<(), EpollHelperError> {
+        Ok(())
+    }
+
+    fn event_list(
+        &mut self,
+        _helper: &mut EpollHelper,
+        _events: &[epoll::Event],
+    ) -> Result<(), EpollHelperError> {
+        Ok(())
+    }
 }
```

Comment:
At this point I think the methods on this trait need explanatory documentation.

Reply:
Sure!